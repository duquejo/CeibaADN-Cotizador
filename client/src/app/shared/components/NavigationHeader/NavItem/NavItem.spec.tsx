import * as React from 'react';
import { ShallowWrapper, mount, render, shallow } from 'enzyme';
import { NavItem } from './index';

describe('Pruebas sobre componente <NavList />', () => {

    let componentWrapper: ShallowWrapper;
    let componentProps: React.ComponentProps<typeof NavItem>;

    beforeEach( () => {
        componentProps = {
            label: 'Item menú 1',
            to: '/menu1'
        };
        componentWrapper = shallow(<NavItem { ...componentProps } />);
    });

    it('Debería renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });
});