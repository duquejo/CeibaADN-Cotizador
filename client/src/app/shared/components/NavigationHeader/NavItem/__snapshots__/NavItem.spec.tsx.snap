// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <NavList /> Debería renderizar el elemento 1`] = `
<styled.div>
  <Styled(Link)
    replace={true}
    to="/menu1"
  >
    Item menú 1
  </Styled(Link)>
</styled.div>
`;
